### DISCUSSION TITLE
谁能告诉我怎么换头像啊啊啊！

### DISCUSSION THREAD

**Discussion by zytcw** [share121/inter-knot#4515] [其他]
刚进来，一头雾水，啥都不会。(而且这个翻译看的好累啊～)

**Comment by duoduo13123678** [comment]
打开 https://github.com/settings/profile，往下滑找到现在的头像，点击左下的edit，上传就可以了
![Screenshot_2024-09-07-07-06-57-998_mark via-edit](https://github.com/user-attachments/assets/609d776c-218e-4717-aa06-daa1d0191221)

**Reply by zytcw** [reply]
好的谢谢
